package.json

```diff
@@ -48,10 +48,13 @@
     ]
   },
   "dependencies": {
+    "@types/mock-fs": "^3.6.30",
```

Comment:
Keep type definitions in `devDependencies`; run `yarn remove @types/mock-fs` and then `yarn add -D @types/mock-fs`